lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -32,7 +32,14 @@
   name:
     description:
       - Name of the security group.
-    required: true
+      - One of and only one of I(name) or I(group_id) is required.
```

Comment:
So I'd change this description here to required when I(state=present) or something.